index.html

```diff
@@ -1169,19 +1169,77 @@ <h2>Relative DID URLs</h2>
 
   </section>
 
-  <section class="informative">
+  <section>
     <h1>Data Model</h1>
-
+    <p>
+This specification defines an abstract data model for <a>DID documents</a>,
+independent of any specific representation. This section provides a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification. This definition MUST enable lossless
+conversion of the value of any property into another representation. This means
```

Comment:
```suggestion
A representation MUST define an unambiguous encoding and decoding of all
properties and their associated values as defined in this specification. This means
```

This language is a bit ambiguous... for example, let's say one format supports 64-bit integers and the other is only capable of supporting 32-bit integers. Does that mean that any format that can only support 32-bit integers is not allowed?

It feels like the first sentence is fine, and the second sentence is not necessary... because if you can unambiguously encode/decode all properties in the spec to/from the data model to the representation and back, then you should be able to go to any other representation w/o issue. Perhaps adding "unambiguously and losslessly" or just "losslessly" is all that's needed in the first sentence?

Reply:
We cannot make normative statements about what representation definitions MUST contain, because those are constraints on a spec rather than implementations (and thus untestable).  I faced the same challenge in the WebRTC work regarding extensibility of the spec and ultimately wrote the language you see [here](https://www.w3.org/TR/mediacapture-streams/#extensibility).  It might be helpful.

Reply:
Yes, let's please do what @burnburn did for WebRTC. That would make me feel much better about the large number of non-testable normative statements we have in the spec.

Reply:
I'm fine with the change, but when I read the WebRTC spec section Dan cited, I was not clear what that meant for this section. I hope we can discuss on the next Editor's call.